### PR TITLE
Preparing release 2.0

### PR DESCRIPTION
**Pull request by leplatrem**
**Step 1**
```
$ git co -b prepare-X.X
$ prerelease
$ vim docs/conf.py
$ git ci -a --amend
$ git push origin prepare-X.X
```

* [x] Merge remaining pull requests
* [x] Update changelog
* [x] Update version in docs/conf.py
* [x] Version dependencies

**Step 2**
```
$ git co master
$ git merge --no-ff prepare-X.X
$ release
$ postrelease
```
* [x] Tag vX.Y.Z
* [x] Publish on Pypi

**Step 3**
* [x] Close milestone in Github
* [x] Add entry in Github release page
* [x] Create next milestone in Github
* [ ] Send mail to ML

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,8 +3,8 @@ Changelog
 
 This document describes changes between each past release.
 
-1.7.1 (unreleased)
-------------------
+2.0 (2015-07-13)
```
Comment (Natim) [inline CHANGELOG.rst]: 2.0.0

File: setup.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
     ]}
 
 setup(name='readinglist',
-      version='1.7.1.dev0',
+      version='2.0',
```
Comment (Natim) [inline setup.py]: 2.0.0

**Comment by Natim** [comment]
'waitress>=0.8.9',
     'cliquet[postgresql,monitoring]>=2.3',

Should be ``==`` instead of ``>=`` for the tag.

**Comment by leplatrem** [comment]
Thanks :)

**Comment by ametaireau** [comment]
cool.

**Pull request closed without merging**